package.json

```
@@ -1120,6 +1120,18 @@
                     "command": "aws.ec2.openTerminal",
                     "when": "aws.isDevMode"
                 },
+                {
+                    "command": "aws.ec2.startInstance",
+                    "whem": "aws.isDevMode"
```

typo (also other cases below). Not sure this is necessary though, unless these commands are "global"

```suggestion
                    "when": "aws.isDevMode"
```

---

still present in other cases